components/AboutMeCards.tsx

```diff
@@ -17,438 +28,196 @@ import {
 import { dosis } from "@/fonts/Dosis";
 import { openSans } from "@/fonts/OpenSans";
 
-export default function AbouMeCardsUp() {
+const cardsData = [
```

Comment:
This kind of data consts must go to the @/constants folder and be imported afterwards in the current file. I'll approve the pull request but we do this together next week, ok?